web/themes/custom/usagov/scripts/benefitSearch.js

```diff
@@ -0,0 +1,440 @@
+    myself.form.prepend(elt.content);
+  };
+  /**
+   * @param {Element} page
+   */
+  this.showPage = function (page) {
+    page.classList.add('page-active');
+    page.classList.remove('display-none');
+  };
+  /**
+   * Display content matching selected categories
+   */
+  this.showResults = function() {
+    myself.resultsContainer.innerHTML = '';
+    // keep the benefits that match
+    let matches = this.findMatches();
+
+    const pages = myself.preparePages(matches);
+    for (const page of pages.map(myself.renderPage)) {
+      myself.resultsContainer.innerHTML += page.outerHTML;
+    }
+
+    if (myself.activePage > pages.length) {
+      myself.setActivePage(pages.length);
+    }
+    else if (myself.activePage < 1) {
+      myself.setActivePage(1);
+    }
+    myself.resultsContainer.scrollIntoView({"behavior": 'smooth'});
+    myself.showPager(pages.length);
+  };
+  /**
+   * Display the pager widget below the results
+   * @param maxPages
+   */
+ this.showPager = function(maxPages) {
+   const pager = new Pagination(
+     maxPages,
+     myself.activePage,
+     myself.assetBase,
+     myself.labels,
+     myself.handlePagerClick
+   );
+   let existing = resultsContainer.querySelector('nav.usa-pagination');
+   if (existing) {
+     existing.remove();
+     resultsContainer.append(pager.render());
+     return;
+   }
+
+   resultsContainer.append(pager.render());
+ };
+  /**
+   * Saves the selected terms and current page to the browser's history via query string
+   */
+  this.updateHistory = function() {
+    const terms = myself.terms.filter((num) => {return false === isNaN(num);});
+    // update query string
+    const url = new URL(window.location.href);
+    url.searchParams.set('t', terms.join('-'));
+    url.searchParams.set('pg', myself.activePage);
+    // update browser
+    window.history.pushState(null, '', url.toString());
+  };
+  /**
+   * Loads the data file and adds event listeners
+   * @returns {Promise<void>}
+   */
+  this.init = async function() {
+    // load data and initial URL state
+    this.benefits = await myself.fetchBenefits();
+    this.lifeEvents = await myself.fetchLifeEvents();
+
+    this.parseUrlState();
+
+    // checkbox events
+    this.toggleAll.addEventListener('click', myself.handleToggleAll);
+    for (const box of myself.boxes) {
+      box.addEventListener('click', myself.handleToggleCheck);
+    }
+    // form events
+    myself.form.addEventListener('submit', myself.handleSubmit);
+    myself.form.addEventListener('reset', myself.handleClear);
+    // history events
+    window.addEventListener('popstate', myself.parseUrlState);
+  };
+}
+
+jQuery(document).ready(async function () {
+  "use strict";
+  let docLang = [document.documentElement.lang];
+  let benefitsPath, lifeEventsPath, labels;
+  // using relative URL so that this works on static pages
+  // Setup language specific inputs
+  if (docLang[0] === 'en') {
+    benefitsPath = "../benefits-search/en.json";
```

Comment:
The hard-coded paths caught my eye. It makes sense that they're hard-coded! But why the relative links? 
I'm also thinking that we should define a path for files we're using like APIs. It would be neater, in my opinion, and it would forestall the possibility of someone wanting to use "benefits-search" as a page name in the future and hitting some kind of conflict. (In practice, I think at most we'd get a warning from Tome the second time it tried to create that directory, but it seems ... awkward.) 
Isabel put the view output for the federal agencies search under /sites/default/files/, which works but also seems a little weird to me. I'm thinking of something like "_data" and I'm open to options. (So for example, these JSON files might be at paths like  /_data/benefits-search/en.json.)